stylesheets/print.styl

```diff
@@ -0,0 +1,139 @@
+body 
+	width 100% !important
+	margin 0 !important
+	padding 0 !important
+	line-height 1.45
+	font-size 14pt
+*:not(.news-item-default-image):not(.date-flag):not(.icon):not(.cd-ic),
+*:before,
+*:after,
+*:first-letter,
+p:first-line,
+div:first-line,
+blockquote:first-line,
+li:first-line 
+	background transparent !important
+    color #000 !important
+	box-shadow none !important
+	text-shadow none !important
+h1,h2,h3,h4,h5,h6 
+	page-break-after avoid
+h1 
+	font-size 19pt !important
+h2 
+	font-size 17pt !important
+h3 
+	font-size 15pt !important
+h4,h5,h6 
+	font-size 14pt !important
+p, h2, h3 
+	orphans 3
+	widows 3
+code 
+	font 12pt Courier, monospace
+blockquote 
+	margin 1.2em
+	padding 1em
+	font-size 12pt
+hr 
+    background-color #ccc
+img 
+	float left
+	margin 1em 1.5em 1.5em 0
+	max-width 100% !important
+a 
+	img 
+		border none
+a:link, a:visited 
+	background transparent
+	font-weight 700
+	text-decoration underline
+	color #333
+a:link[href^="http://"]:after, a[href^="http://"]:visited:after 
+	content ""
+a[href]:after 
+	content ""
+a[href^="mailto"]:after 
+	content " (" attr(href) ") "
+	font-size 90%
+a[href^="http 
+	&://"] 
+		color #000
+a[href$=".jpg"]:after, a[href$=".jpeg"]:after, a[href$=".gif"]:after, a[href$=".png"]:after 
+	content " (" attr(href) ") "
+	display none
+a[href^="#"]:after, a[href^="javascript:"]:after 
+	content ""
+table 
+	margin 1px
+	text-align left
+th 
+	border-bottom 1px solid #333
+	font-weight bold
+td 
+	border-bottom 1px solid #333
+th,td 
+	padding 4px 10px 4px 0
+tfoot 
+	font-style italic
+caption 
+    background #fff
+	margin-bottom 2em
+	text-align left
+thead 
+	display table-header-group
+img,tr 
+	page-break-inside avoid
+header 
+	&.h 
+		position absolute
+span 
+	&.lwi-t 
+		width 100% !important
+.node-type-topic-page 
+	.hero-image 
+		border-bottom none
+	.intro-text-top 
+		border-top 0
+		border-left none
+		border-right none
+.lo--abs 
+	left 40px
+.b--by 
+	border-top 0
+.hro-st--l 
+	font-size 15pt !important
+.tab-ctrl-0:checked~.hro .tab-li-0 .tab-li-a,
+.tab-ctrl-0:checked~.tab .tab-li-0 .tab-li-a,
+.tab-ctrl-1:checked~.hro .tab-li-1 .tab-li-a,
+.tab-ctrl-1:checked~.tab .tab-li-1 .tab-li-a,
+.tab-ctrl-2:checked~.hro .tab-li-2 .tab-li-a,
+.tab-ctrl-2:checked~.tab .tab-li-2 .tab-li-a,
+.tab-ctrl-3:checked~.hro .tab-li-3 .tab-li-a,
+.tab-ctrl-3:checked~.tab .tab-li-3 .tab-li-a 
+    color #091f2f
```

Comment:
Rather than a hex, use `$charles-blue` to import the value from the `_colors.styl` definitions.